app/Http/Requests/StoreAppendBundleRequest.php

```diff
@@ -37,4 +38,21 @@ public function rules()
 
         return $rules;
     }
+
+    protected function prepareForValidation()
+    {
+        // get the input and remove null/empty values.
+        // TODO: 5.5 upgrade will need to change this to 'all' with params.
+        $input = array_filter(
+            $this->only(['start', 'end']),
+            'strlen'
```

Comment:
Not too keen on 0 == false but I think it's OK here.

Reply:
it's apparently a well known array_filter trick :man_shrugging: